src/man/sssd-secrets.5.xml

```diff
@@ -107,14 +107,17 @@ systemctl enable sssd-secrets.service
                 <term>provider (string)</term>
                 <listitem>
                 <para>
-                    This option specifies where should the secrets
-                    be stored. The secrets responder can configure a
-                    per-user subsections that define which provider store
-                    the secrets for this particular user. The per-user
-                    subsections should contain all options for that user's
-                    provider. If a per-user section does not exist, the
-                    global settings from the secret responder's section
-                    are used.  The following providers are supported:
+                    This option specifies where should the secrets be
+                    stored. The secrets responder can configure a per-user
+                    subsections (e.g. [secrets/users/123] - see bottom of
+                    this manual page for a full example using Custodia
+                    for a particular user) that define which provider
```

Comment:
"(...) that define which provider (...)" -> "(...) that defines which provider (...)"